public/app/features/plugins/loader/utils.test.ts

```diff
@@ -0,0 +1,31 @@
+import { config } from '@grafana/runtime';
+
+import { resolveModulePath } from './utils';
+
+describe('', () => {
```

Comment:
Please give a name to this test suite.

Reply:
Ah well spotted. Fixed in https://github.com/grafana/grafana/pull/81658/commits/3dfb34d1a311ce2c8aa147885694d0dcf2072885